tests/data/firefox/tinderbox-builds/mozilla-central-linux/1374583609/README.md

```diff
@@ -0,0 +1,3 @@
+This is supposed to be an empty directory to serve as a test case for a missing binary.
+
+See Issue #143 "TinderboxScraper: Omit folders without binaries"
```

Comment:
@whimboo 

Well this is a specific test for it ;)

For this test: https://github.com/mozilla/mozdownload/blob/master/tests/tinderbox_scraper/test_tinderbox_scraper.py#L46

the empty folder would be chosen, since it is the latest build. But the PR allows it now to check if there is actually something inside the directory first.

I just extended the original test, by adding to the test directory structure.